docs-src/pages/UiTabs.vue

```diff
@@ -73,6 +73,18 @@
                 </ui-tab>
             </ui-tabs>
 
+            <h4 class="page__demo-title">Tabs with tooltips</h4>
+
+            <ui-tabs type="icon">
+                <ui-tab :key="tab.title" v-for="tab in demoTabs" :tooltip="tab.title">
+                    <template #icon>
+                        <ui-icon :icon="tab.icon"></ui-icon>
+                    </template>
```

Comment:
In my local testing, I could only get this to work when changed to
```suggestion
                    <ui-icon slot="icon" :icon="tab.icon"></ui-icon>
```

Reply:
Looks like your fork is still on Vue 2, that's why it wasn't working. It's fine when merged with `next` which has Vue 3.